src/graphql/operations/leaderboards.ts

```diff
@@ -8,31 +9,35 @@ export default async function (parent, args) {
 
   checkLimits(args, 'leaderboards');
 
+  const ORDER_FIELDS = ['vote_count', 'proposal_count', 'user'];
+  const DEFAULT_ORDER_FIELD = 'vote_count';
+
   const fields = {
     user: 'EVMAddress',
     space: 'string',
     vote_count: 'number',
     proposal_count: 'number'
   };
   const whereQuery = buildWhereQuery(fields, 'l', where);
-  const defaultOrder = 'votesCount DESC, proposalsCount DESC';
 
-  const orderBy = Object.keys(fields).includes(args.orderBy)
+  const orderBy = ORDER_FIELDS.includes(args.orderBy)
     ? args.orderBy
-    : null;
+    : DEFAULT_ORDER_FIELD;
   let orderDirection = (args.orderDirection || 'desc').toUpperCase();
   if (!['ASC', 'DESC'].includes(orderDirection)) orderDirection = 'DESC';
 
+  const orderQuery = uniq([orderBy, ...ORDER_FIELDS])
+    .map((field: string) => `l.${field} ${orderDirection}`)
+    .join(', ');
+
   const query = `
     SELECT l.*,
       l.vote_count as votesCount,
       l.proposal_count as proposalsCount,
       l.last_vote as lastVote
```

Comment:
Sorry for not noticing it earlier, but I think it is not very clear that we use underscore in inputs and camelCase on outputs, wyt? 🤔 

because `orderBy` is usually `string` and we may pick it from the output schema

Reply:
also input is `vote` count while output is `votes` count

Reply:
I use `proposalsCount` as output, to follow the convention in Space. It is camel case, and plural

Reply:
We have the same issue with `User`. We use `proposal_count` in where and order, and it return `votesCount`.

Not sure why we’re using a mix of underscore and camel case for output, but I just followed what we’re already using for counter fields